hack/verify-idl-consumption.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Usage: $0
+# Current working directory needs to be the repo root.
+# The git status is assumed to be clean at the start.
+# This script checks that the IDL processing is correct.
+
+rm -rf config/crd $(find api -name "zz_generated*")
+make manifests generate
```

Comment:
It's recommended to add error checking:

`make manifests generate || { echo "Make failed"; exit 1; }
`